apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultsTable.tsx

```diff
@@ -62,7 +62,13 @@ export const ResultCellContent = ({
     )
   }
 
-  return <Text.H4 noWrap>{value as string}</Text.H4>
+  return (
+    <Text.H4 noWrap>
+      {typeof value === 'string' && value.length > 30
+        ? `${value.slice(0, 30)}...`
+        : (value as string)}
```

Comment:
Wait. This should not be necessary. Review the CSS it can be done with ellipsis prop in this text + a wrap div. Look for text examples with ellipsis

Reply:
i think it's because it's a table element

Reply:
what i mean is that in order for it to work i would have to put a max width limit to the wrapper div which ends up being the same hardcoding